velox/functions/prestosql/aggregates/MinMaxAggregates.cpp

```diff
@@ -280,8 +280,11 @@ const T MinAggregate<T>::kInitialValue_ = MinMaxTrait<T>::max();
 
 class NonNumericMinMaxAggregateBase : public exec::Aggregate {
  public:
-  explicit NonNumericMinMaxAggregateBase(const TypePtr& resultType)
-      : exec::Aggregate(resultType) {}
+  explicit NonNumericMinMaxAggregateBase(
+      const TypePtr& resultType,
+      bool shouldCheckNestedNull)
```

Comment:
nit: perhaps, throwOnNestedNulls and throwOnNestedNulls_

Reply:
Fixed